playbooks/roles/edxapp/tasks/deploy.yml

```
@@ -357,13 +357,25 @@
   with_items:
     - "lms.sh"
     - "cms.sh"
+    - "scheduler.sh"
```

```suggestion
    - "beat_scheduler.sh"
```
I'd prefer that we rename this so that it's more obvious what it is to someone who's not aware of or thinking about celery beat.